Lib/datetime.py

```diff
@@ -2269,8 +2269,8 @@ def fromutc(self, dt):
         raise TypeError("fromutc() argument must be a datetime instance"
                         " or None")
 
-    _maxoffset = timedelta(hours=23, minutes=59)
-    _minoffset = -_maxoffset
+    _maxoffset = timedelta(hours=23, minutes=59, seconds=59, microseconds=999999)
```

Comment:
```suggestion
    _maxoffset = timedelta(hours=23, minutes=59, seconds=59,
                           microseconds=999999)
```

Line is now too long. Either use this suggestion or @joernheissler's suggestion of `timedelta(hours=24, microseconds=-1)`. I think there are upsides and downsides to both ways of expressing this concept, so I'll leave it in your hands which you like better.